packages/strapi-admin/admin/src/validations/users/profile.js

```diff
@@ -23,6 +23,7 @@ const schema = {
     .when('password', (password, passSchema) => {
       return password ? passSchema.required(translatedErrors.required) : passSchema;
     }),
+  preferedLanguage: yup.string().nullable(),
```

Comment:
ℹ️ It's nullable because people can eventually modify their password while not having preferred language saved. Let me know if we have to put this in required.